filebeat/module/traefik/access/config/traefik-access.yml

```diff
@@ -4,3 +4,12 @@ paths:
  - {{$path}}
 {{ end }}
 exclude_files: [".gz$"]
+
+processors:
+- dissect:
+    tokenizer: '%{traefik.access.remote_ip} - %{traefik.access.user_name} [%{traefik.access.time}]
+    "%{traefik.access.method} %{traefik.access.url} HTTP/%{traefik.access.http_version}"
+    %{traefik.access.response_code} %{traefik.access.message}'
```

Comment:
@ruflin  I think this is a good first step, after looking at the complete if dissect was support chaining of tokenizer we could probably extract everything with 2 dissect processor.